index.html

```diff
@@ -14,6 +14,64 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+<!------------------------------------header------------------------------>
+    <header>
+        <div class="logo">
+            <img src="img/karma-logo.svg" id="left-logo" alt="logo">
+        </div>
+        <div class="list">
+            <ul>
+                <li>Meet Karma</li>
+                <li>How it works</li>
+                <li><a href="#">store</a></li>
+                <li><a href="#">blog</a></li>
+                <li><a href="#">help</a></li>
+                <li><a href="#">login</a></li>
+            </ul>
+        </div>
+    </header>
+<!-----------------------------------main area------------------------------>
+    <main>
+        <div class="karma">
+            <img src="img/first-background.jpg" alt="photo">
+            <div class="details">
+                <h1>introducing karma</h1>
+                <p>Bring WIFI with you, everywhere you go.</p>
+                <button>learn more</button>
+            </div>
+        </div>
+    </main>
+<!------------------------------------section-------------------------------->
+ 
+<section>
+    <article class="karma1">
+        <h1>Everyone needs a little Karma.</h1>
+        <div class="karma-img">
+            <div>
+                <img src="img/icon-devices.svg" alt="photo">
+                <p>Internet for all devices</p>
+            </div>
+            <div>
+                <img src="img/icon-coffee.svg" alt="photo">
+                <p>Boost your productivity</p>
+            </div>
+            <div>
+                <img src="img/icon-refill.svg" alt="photo">
+                <p>Pay as You Go</p>
+            </div>
+        </div>
 
+    </article>
+</section>
+<!----------------------------------------footer------------------------------------------>
+<footer>
+    <h4>Join us on</h4>
+    <div class="links">
+        <a href="#"><img src="img/twitter-icon.svg" alt="photo"></a>
+        <a href="#"><img src="img/facebook-icon.svg" alt="photo"></a>
+        <a href="#"><img src="img/instagram-icon.svg" alt="photo"></a>
+    </div>
+    <h5>&copy; karma mobility.inc</h5>
+</footer>
```

Comment:
Hi, by the look of your CSS, everything seems good. Could you please don't forget at Media QUeries "Responsive web-design technique"

Reply:
Remember for this project we were all required to just design the desktop view, not with @media query that's why it's sort of missing, anyway thanks for mentioning it, much appreciated